functions/consumer/xmpp-consumer/src/main/java/org/springframework/cloud/fn/consumer/xmpp/XmppConsumerProperties.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.fn.consumer.xmpp;
+
+import jakarta.validation.constraints.NotEmpty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+/**
+ *
+ * @author Daniel Frey
+ * @since 4.0.0
+ */
+@ConfigurationProperties("xmpp.consumer")
+@Validated
+public class XmppConsumerProperties {
+
+	/**
+	 * XMPP handle to send message to.
+	 */
+	private String chatTo;
+
+	/**
```

Comment:
We typically do not add javadoc on the setters but instead use the config props description on the field to be the documenter.

Reply:
I'll remove the javadocs. I was following what we previously did.

Reply:
@dmfrey ,

Not sure how others, but I prefer to have changes applied without confirmation via comments.
Otherwise it sounds like a spam in my email box: first commit with the change we have requested then your comments that you have done them 😄

Reply:
I understand wanting to "ack" the requested change via the "this thing is done" but it **can** get spammy on notifications. A couple of less-spammy alternatives that achieve the same purpose are: 
1. when complete is just adding the thumbsup emoji as I don't think that sends a notification (if it does send notification then it would be no better of course and would come off of this list)
2. just resolving the comment - that also sends the signal that it is complete. 
3. summary reply message when "all the requested changes are in".